pkg/common/configs/configvalidator.go

```diff
@@ -76,6 +76,9 @@ func checkResource(res map[string]string) (int64, error) {
 	var totalres int64
 	for _, val := range res {
 		rescount, err := strconv.ParseInt(val, 10, 64)
+		if rescount < 0 {
+			return 0, fmt.Errorf("not valid negative resource found")
+		}
```

Comment:
Nit: Logical would be to first check for a parse error and then the value set after a parse error